app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -265,6 +267,29 @@ a.inplace-editing--trigger-link,
     &.edit-strategy-textarea
       margin-top: auto
 
+// Explicit styles for comment / input-esque trigger appearance
+.work-packages--activity--add-comment
+  margin: 20px 0
+
+  .inplace-editing--container
+    border: 1px solid #eee
+
+  .inplace-edit--read-value
+    padding: 10px 5px
+
+  .inplace-editing--trigger-link
+    color: #888
+    font-size: 1rem
+
+  // Disabled comment submit styles when empty
+  .inplace-edit--control--save[disabled],
+  .inplace-edit--control--send[disabled]
+    background-color: $inplace-edit--bg-color--disabled
```

Comment:
I'll move it, even though I can't address the styling for other non-empty fields in this PR.